Variable substitution in docker-compose.yaml makes `oc import docker-compose ...` error out

Hi,

So I am using this [docker-compose.yaml](https://github.com/containscafeine/helloworld-msa-compose/blob/master/docker-compose.yaml) file, which contains services described as such

```yaml
services:
  hello:
    image: redhatmsa/hello:latest
    ports:
    - "${HELLO_PORT}:8080"
    labels:
     - "app=hello"
     - "hystrix.enabled=true"
```

I have `HELLO_PORT` environment variable set in my environment, which is what is substituted in this file when I deploy using Docker Compose.

However, OpenShift errors out when I run `# oc import docker-compose -f docker-compose.yaml`

This issue goes away when I hard-code the values of environment variables in the `docker-compose.yaml` file.

##### Version
```bash
# oc version
oc v1.3.0-alpha.1-20-g30f598e-dirty
kubernetes v1.3.0-alpha.1-331-g0522e63
```

##### Steps To Reproduce
1. `wget https://raw.githubusercontent.com/containscafeine/helloworld-msa-compose/master/docker-compose.yaml`
2. `oc import docker-compose -f docker-compose.yaml`

##### Current Result

```bash
# oc import docker-compose -f docker-compose.yaml
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xb code=0x1 addr=0x20 pc=0x114b9b6]

goroutine 1 [running]:
github.com/openshift/origin/third_party/github.com/docker/libcompose/project.func·001(0xc2084b82b8, 0x8, 0x0, 0x0)
	/root/go/src/github.com/openshift/origin/_output/local/go/src/github.com/openshift/origin/third_party/github.com/docker/libcompose/project/interpolation.go:155 +0x86
github.com/openshift/origin/third_party/github.com/docker/libcompose/project.parseVariableWithBraces(0xc20853f790, 0x10, 0xa, 0xc20875c750, 0x0, 0x0, 0x7fd700000000, 0xc2084dbb90)
	/root/go/src/github.com/openshift/origin/_output/local/go/src/github.com/openshift/origin/third_party/github.com/docker/libcompose/project/interpolation.go:54 +0x161
github.com/openshift/origin/third_party/github.com/docker/libcompose/project.parseInterpolationExpression(0xc20853f790, 0x10, 0x1, 0xc20875c750, 0x0, 0x0, 0x0, 0x30)
	/root/go/src/github.com/openshift/origin/_output/local/go/src/github.com/openshift/origin/third_party/github.com/docker/libcompose/project/interpolation.go:72 +0xa7
github.com/openshift/origin/third_party/github.com/docker/libcompose/project.parseLine(0xc20853f790, 0x10, 0xc20875c750, 0x0, 0x0, 0x1cd9201)
	/root/go/src/github.com/openshift/origin/_output/local/go/src/github.com/openshift/origin/third_party/github.com/docker/libcompose/project/interpolation.go:91 +0xce
github.com/openshift/origin/third_party/github.com/docker/libcompose/project.parseConfig(0xc20853f780, 0x5, 0xc20853f749, 0x3, 0xc20875c508, 0xc20875c750, 0x0, 0x0)
	/root/go/src/github.com/openshift/origin/_output/local/go/src/github.com/openshift/origin/third_party/github.com/docker/libcompose/project/interpolation.go:111 +0x513
github.com/openshift/origin/third_party/github.com/docker/libcompose/project.parseConfig(0xc20853f780, 0x5, 0xc20853f749, 0x3, 0xc2084b8290, 0xc20875c750, 0x0, 0x0)
	/root/go/src/github.com/openshift/origin/_output/local/go/src/github.com/openshift/origin/third_party/github.com/docker/libcompose/project/interpolation.go:128 +0x1ad
github.com/openshift/origin/third_party/github.com/docker/libcompose/project.interpolate(0x0, 0x0, 0xc208670278, 0x0, 0x0)
	/root/go/src/github.com/openshift/origin/_output/local/go/src/github.com/openshift/origin/third_party/github.com/docker/libcompose/project/interpolation.go:168 +0x1eb
github.com/openshift/origin/third_party/github.com/docker/libcompose/project.mergeProject(0xc2086e2480, 0xc2084079e0, 0x29, 0xc2086dd000, 0xc56, 0xe56, 0xc20853f5e8, 0x0, 0x0)
	/root/go/src/github.com/openshift/origin/_output/local/go/src/github.com/openshift/origin/third_party/github.com/docker/libcompose/project/merge.go:59 +0x228
github.com/openshift/origin/third_party/github.com/docker/libcompose/project.(*Project).load(0xc2086e2480, 0xc2084079e0, 0x29, 0xc2086dd000, 0xc56, 0xe56, 0x0, 0x0)
	/root/go/src/github.com/openshift/origin/_output/local/go/src/github.com/openshift/origin/third_party/github.com/docker/libcompose/project/project.go:127 +0xad
github.com/openshift/origin/third_party/github.com/docker/libcompose/project.(*Project).Parse(0xc2086e2480, 0x0, 0x0)
	/root/go/src/github.com/openshift/origin/_output/local/go/src/github.com/openshift/origin/third_party/github.com/docker/libcompose/project/project.go:67 +0x35a
github.com/openshift/origin/pkg/generate/dockercompose.Generate(0xc2085436d0, 0x1, 0x1, 0xc2086700f0, 0x0, 0x0)
	/root/go/src/github.com/openshift/origin/_output/local/go/src/github.com/openshift/origin/pkg/generate/dockercompose/generate.go:53 +0x4d7
github.com/openshift/origin/pkg/cmd/cli/cmd/importer.(*DockerComposeOptions).Run(0xc2082e3400, 0x0, 0x0)
	/root/go/src/github.com/openshift/origin/_output/local/go/src/github.com/openshift/origin/pkg/cmd/cli/cmd/importer/dockercompose.go:146 +0x68
github.com/openshift/origin/pkg/cmd/cli/cmd/importer.func·001(0xc208555c00, 0xc2083f3b20, 0x0, 0x2)
	/root/go/src/github.com/openshift/origin/_output/local/go/src/github.com/openshift/origin/pkg/cmd/cli/cmd/importer/dockercompose.go:83 +0xcf
github.com/spf13/cobra.(*Command).execute(0xc208555c00, 0xc2083f3800, 0x2, 0x2, 0x0, 0x0)
	/root/go/src/github.com/openshift/origin/Godeps/_workspace/src/github.com/spf13/cobra/command.go:572 +0x82f
github.com/spf13/cobra.(*Command).ExecuteC(0xc20829ba00, 0xc208555c00, 0x0, 0x0)
	/root/go/src/github.com/openshift/origin/Godeps/_workspace/src/github.com/spf13/cobra/command.go:662 +0x4db
github.com/spf13/cobra.(*Command).Execute(0xc20829ba00, 0x0, 0x0)
	/root/go/src/github.com/openshift/origin/Godeps/_workspace/src/github.com/spf13/cobra/command.go:618 +0x3a
main.main()
	/root/go/src/github.com/openshift/origin/_output/local/go/src/github.com/openshift/origin/cmd/oc/oc.go:27 +0x175

goroutine 5 [chan receive]:
github.com/golang/glog.(*loggingT).flushDaemon(0x305e140)
	/root/go/src/github.com/openshift/origin/Godeps/_workspace/src/github.com/golang/glog/glog.go:879 +0x78
created by github.com/golang/glog.init·1
	/root/go/src/github.com/openshift/origin/Godeps/_workspace/src/github.com/golang/glog/glog.go:410 +0x2a7

goroutine 17 [syscall, locked to thread]:
runtime.goexit()
	/usr/local/go/src/runtime/asm_amd64.s:2232 +0x1

goroutine 13 [syscall]:
os/signal.loop()
	/usr/local/go/src/os/signal/signal_unix.go:21 +0x1f
created by os/signal.init·1
	/usr/local/go/src/os/signal/signal_unix.go:27 +0x35

goroutine 50 [runnable]:
net/http.(*persistConn).readLoop(0xc2086276b0)
	/usr/local/go/src/net/http/transport.go:928 +0x9ce
created by net/http.(*Transport).dialConn
	/usr/local/go/src/net/http/transport.go:660 +0xc9f

goroutine 51 [select]:
net/http.(*persistConn).writeLoop(0xc2086276b0)
	/usr/local/go/src/net/http/transport.go:945 +0x41d
created by net/http.(*Transport).dialConn
	/usr/local/go/src/net/http/transport.go:661 +0xcbc
```

##### Expected Result
Application gets deployed successfully

--- a/pkg/generate/dockercompose/generate.go
+++ b/pkg/generate/dockercompose/generate.go
@@ -50,6 +50,9 @@ func Generate(paths ...string) (*templateapi.Template, error) {
 		ComposeFiles: paths,
 	}
 	p := project.NewProject(context)
+	if err := project.AddEnvironmentLookUp(context); err != nil {
+		return nil, err
+	}
 	if err := p.Parse(); err != nil {
 		return nil, err
 	}

